meta-error-handling/static_assert.md

```diff
@@ -0,0 +1,113 @@
+# Module name: `static_assert`
+
```

Comment:
```suggestion
_Skeleton instructions are typeset in italic text._

```

Reply:
Is it the recommended practice to keep the skeleton instructions in the final document? I've been removing them as I fill in the topic

Reply:
I don't believe the original intention was so, but it might be a good idea since it will provide an outline for people editing the document down the road.